### PR TITLE
7279 change drop down menu

### PR DESCRIPTION
**Pull request by asanametove**

### CODE REVIEW

File: conference/react-js/src/components/Tabs/Tabs.jsx [hunk 1]
```diff
@@ -1,53 +1,126 @@
...
-        component={MyEvents}
-      />
-      <Route
-        path={`${conference}/:id`}
-        exact
-        component={Conference}
-      />
+      <NavigationBar>
+        {
+          getLinks(talksCount, conferenceCount, role)
```
Comment (ScreamCat1) [inline conference/react-js/src/components/Tabs/Tabs.jsx]: I think, you can write map function with add NavLink inside getLinks

File: conference/react-js/src/components/UserMenu/UserMenu.jsx [hunk 2]
```diff
@@ -1,28 +1,13 @@
...
-    <Link
-      className="menu-list__title"
-      to={link}
-      onClick={click}
-    >
-      {title}
-    </Link>
-  </li>
-));
+import MenuItem from './MenuItem';
```
Comment (ScreamCat1) [inline conference/react-js/src/components/UserMenu/UserMenu.jsx]: Are you sure about we need this component? We used it only once, and you can can write this easier (use li and Link with className to sign-in and out)

File: conference/react-js/src/containers/ManageUsers/ManageUsers.jsx [hunk 3]
```diff
@@ -35,40 +35,27 @@ class ManageUser extends PureComponent {
...
-          </ul>
-          <div className="tabs-container">
-            <div className="my-talks__header">
-              <button className="btn my-talks__button">
-             Add New User
-              </button>
+      <div className="tabs-container">
+        <div className="my-talks__header">
+          <button className="btn my-talks__button">
+          Add New User
```
Comment (ScreamCat1) [inline conference/react-js/src/containers/ManageUsers/ManageUsers.jsx]: Linter did not fix it. But, i think, You should use more indents when write text

**Review by ScreamCat1 — APPROVED** [review]

File: conference/react-js/src/components/Tabs/Tabs.jsx [hunk 4]
```diff
@@ -1,53 +1,125 @@
...
+        }
+      </NavigationBar>
+      <Switch>
+        <Route
```
Comment (SupraVlad) [inline conference/react-js/src/components/Tabs/Tabs.jsx]: let's extract this list to separate variable. it is getting bigger and bigger seems to me

**Review by SupraVlad — CHANGES_REQUESTED** [review]

**Review by SupraVlad — APPROVED** [review]

**Pull request merged**